src/index.html

```diff
@@ -11,7 +11,209 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    name="forms"
+    >
+
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div class="lastName">
+          <label for="lastName1">
+            Surname:
+            <input
+            type="text"
+            name="lastName"
+            id="lastName1"
```

Comment:
Remove these `1`'s from all of your id's
```suggestion
            id="lastName"
```